lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": ["packages/*"],
```

Comment:
This isn't needed with `useWorkspaces: true`, `package.json` `workspaces` are used instead.